src/Orders/OrderScreen.js

```diff
@@ -0,0 +1,180 @@
+/* eslint-disable no-plusplus */
+import React, { useEffect, useState } from 'react';
+import {
+  View, StyleSheet, ScrollView, Alert,
+} from 'react-native';
+import { Title } from 'react-native-paper';
+import PropTypes from 'prop-types';
+import Config from 'react-native-config';
+import OrderCard from './OrdersCard';
+
+const Airtable = require('airtable');
+
+const airtableConfig = {
+  apiKey: Config.REACT_APP_AIRTABLE_USER_KEY,
+  baseKey: Config.REACT_APP_AIRTABLE_BASE_KEY,
+};
+
+const base = new Airtable({ apiKey: airtableConfig.apiKey })
+  .base(airtableConfig.baseKey);
+
+const CONST_USER = 'helen@gmail.com';
+const MONTHS = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"];
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: '#FFFFFA',
+  },
+  centeredContainer: {
+    marginTop: '10%',
+  },
+  titleText: {
+    marginBottom: 20,
+    fontSize: 30,
+    marginStart: 20,
+    fontFamily: 'JosefinSans-SemiBold',
+  },
+  subtitleText: {
+    marginBottom: 10,
+    fontSize: 20,
+    marginStart: 27,
+    marginTop: 10,
+    fontFamily: 'JosefinSans-SemiBold',
+    color: '#1D763C',
+  },
+  orderCardsContainer: {
+    display: 'flex',
+    flexDirection: 'column',
+    flexWrap: 'wrap',
+    alignContent: 'center',
+    justifyContent: 'space-around',
+  },
+});
+
+export default function OrderScreen({ navigation }) {
+  const [orderMap, setOrderMap] = useState(new Map());
+  const [itemsList, setItemsList] = useState(new Map());
+  const [images, setImages] = useState(new Map());
+  const [totalItemsList, setTotalItemsList] = useState(new Map());
```

Comment:
does this ever get used again? if not, delete totalItemsList from the state variables

Reply:
done